.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @suho @malparty
```

Comment:
Let's add `markgravity` as well, his contributions will help the project move forward 🦄  

...and maybe also an empty line to end the file 😇